src/ZPublisher/tests/testHTTPResponse.py

```diff
@@ -1243,8 +1243,8 @@ def test___str__after_expireCookie(self):
         response.expireCookie('qux', path='/')
         result = bytes(response)
         lines = result.split(b'\r\n')
-        cookie_line = [l for l in lines if b'Set-Cookie' in l][0]
-        other_lines = [l for l in lines if b'Set-Cookie' not in l]
+        cookie_line = [x for x in lines if b'Set-Cookie' in x][0]
+        other_lines = [x for x in lines if b'Set-Cookie' not in x]
```

Comment:
While `l`is ambiguous, `x` is better readable, but still non descriptive.

What about changing it to `line`?

Reply:
Okay